src/inc/til/point.h

```diff
@@ -229,6 +240,19 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
             return wil::str_printf<std::wstring>(L"(X:%d, Y:%d)", x, y);
         }
     };
+
+    constexpr point wrap_coord(const COORD rect) noexcept
+    {
+        return { rect.X, rect.Y };
+    }
+
+    constexpr COORD unwrap_coord(const point rect)

```

Comment:
I almost would have preferred `wrap_COORD` and `unwrap_COORD` so we knew they were talking about the type. Theoretically, they could mean any type of coord right now. Eh.

Reply:
We'll only have about a few dozen calls to these wrap/unwrap functions throughout the entire project in the future in mostly just two files (propsheet / console API dispatcher).
So it should be easy to rename them in the future. I prefer the lowercase name personally for no specific reason apart from how it looks.